config/routes.rb

```diff
@@ -12,6 +12,7 @@
 
   get '/sign_up', to: 'registrations#new', as: 'sign_up'
   get '/log_in', to: 'sessions#new', as: :log_in
+  get '/reviews', to: 'reviews#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.